gap/small.gd

```diff
@@ -80,6 +80,40 @@ UnbindGlobal( "SMALL_AVAILABLE" );
 ##
 DeclareGlobalFunction( "SMALL_AVAILABLE" );
 
+#############################################################################
+##
+#F  SmallGroupsAvailable( <order> )
+##
+##  <#GAPDoc Label="SmallGroupsAvailable">
+##  <ManSection>
+##  <Func Name="SmallGroupsAvailable" Arg='order'/>
+##
+##  <Description>
+##  returns <C>true</C> if the library of groups of <A>order</A> is installed, and
```

Comment:
In my opinion this sounds strange: substituting 6 for `<order>`, we get `"...if the library of groups of 6 is installed"`. The same is true for the other bits of documentation. The solution would be to write `"...if the library of groups of order <A>order</A> is installed"`, which I realise is repetitive. So I wouldn't complain if you kept it how it is – I think it is suitably clear.